tests/sm_int/sm_int.c

```diff
@@ -410,19 +379,19 @@ typedef struct THREADS_COMMON_TAG
 
 MU_DEFINE_ENUM(THREAD_TYPE, THREAD_TYPE_VALUE)
 
-static DWORD WINAPI barrier_thread(
-    LPVOID lpThreadParameter
+static int barrier_thread(
+    void* lpThreadParameter
```

Comment:
>lpThreadParameter [](start = 10, length = 17)

nit: lpThreadParameter sounds like remnants of win32 API

Reply:
Renamed to arg

---
In reply to: [516819700](https://github.com/Azure/c-util/pull/75#discussion_r516819700) [](ancestors = 516819700)